VERSION

```diff
@@ -0,0 +1 @@
+v3.2.2
```

Comment:
How about setting version info in a `version.h`?
```suggestion
#ifndef VERSION
#define VERSION "v3.2.2"
#endif
```

Reply:
Currently, it is passed using macros. What are the benefits of placing it in the version. h file?

Reply:
> Currently, it is passed using macros. What are the benefits of placing it in the version. h file?

Just by common practice. It's OK to keep your original implementation.

Reply:
Ok, I accept your suggestion.